### DISCUSSION TITLE
Why are two-factor authentications not working on my new phone after uninstalling from my old phone?

### DISCUSSION THREAD

**Discussion by saipirakib** [community/community#34918] [Other Feature Feedback, Questions, & Ideas]
I had not used my second phone where I installed the two-factor authentication app and it was working. But uninstall this app and install it on my new phone. But this app can't generate any password for 2FA in Github login. Please help me. I tried a lot to solve this problem. Or manually of these two-factor authentication options. My email saifrakib1@gmail is associated with Google plus Github login. saifrakib1@gmail.com This was an official email but I can't use this email in my GitHub log in that case I had to create a new one to solve this problem. I import all the accounts from my previous phone but 2FA apps don't generate any password for GitHub or any other apps
![two factor email](https://user-images.githubusercontent.com/114886748/193504309-5a9b37db-8c18-474a-a968-5fec7a80fadf.png)

**Comment by hariprasd** [comment]
I hope you had imported the 2 factor auth app to your new mobile.

**Reply by saipirakib** [reply]
Yeah I did, even though I can't generate any codes
